src/components/Header/ActionsMenu.scss

```diff
@@ -0,0 +1,87 @@
+.user-actions-menu li {
+  cursor: pointer;
+}
+
+.user-actions-menu a {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+}
+.user-actions-menu a,
+.menu-label {
+  font-size: 1.06rem;
+}
+
+.user-actions-menu svg {
+  margin-right: 10px;
+}
+
+@media only screen and (min-width: 64em) {
+  .user-actions-container {
+    color: #fff;
+  }
+
+  .user-actions-container .menu-label {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+    font-weight: bold;
+    position: relative;
+  }
+
+  .user-actions-menu {
+    display: none;
+    background-color: #0071bc;
```

Comment:
FYI, in my current branch, I've created a new SCSS file that captures all of these MACBIS colors as reusable variables. Figured that could help us stay consistent and make it easy to update any colors if design wants to tweak something. 😅 I'll be happy to walk you through it at stand-up